src/modules/convex/linear_svm_igd.cpp

```diff
@@ -152,9 +144,18 @@ linear_svm_igd_final::run(AnyType &args) {
 
     // Aggregates that haven't seen any data just return Null.
     if (state.algo.numRows == 0) { return Null(); }
+    
```

Comment:
Same as above - I would prefer to have one `if (isL2)` that decides between the L2 and L1 functions.